### ISSUE TITLE
Revise GO:0097250 mitochondrial respirasome assembly 

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-ontology#29227]
GO:0097250 mitochondrial respirasome assembly 
is defined as "The aggregation, arrangement and bonding together of a set of respiratory enzyme complexes of the mitochondrial inner membrane (including, for example, complex II, complex III, complex IV) to form a large supercomplex."

and 
GO:0033617 mitochondrial cytochrome c oxidase assembly
GO:0033615 mitochondrial proton-transporting ATP synthase complex assembly
GO:0032981 mitochondrial respiratory chain complex I assembly
GO:0034551 mitochondrial respiratory chain complex III assembly
GO:0034553 mitochondrial respiratory chain complex II assembly

have a part_of rel

But the 'respirasome' is really a grouping term for supercomplexes formed by CI, CIII and CIV. Specifically for 

PMID: 30030361  and PMID:32311046 say that there are various supercomplexes of CI, CIII, CIV; the stoichiometry 1:2:1 is the 'respirasome', 2:2:2 is termed the ‘respiratory megacomplex’

As there seem to be a handful of factors that are specifically associated with the assembly of these complexes such as human HIGD1A, HIGD2A, RAB5IF, we need this term but with a few changes.

Think that we can keep the name 'mitochondrial respirasome assembly' as there is a fairly loose usage in the literature and tweak the defintion to 
"The aggregation, arrangement and bonding together of respiratory enzyme complexes I, III and IV of the mitochondrial inner membrane to form a large supercomplex."

remove the has_part:
GO:0033617 mitochondrial cytochrome c oxidase assembly
GO:0033615 mitochondrial proton-transporting ATP synthase complex assembly
GO:0032981 mitochondrial respiratory chain complex I assembly
GO:0034551 mitochondrial respiratory chain complex III assembly
GO:0034553 mitochondrial respiratory chain complex II assembly
(if we want to group these with a parent, we should make a grouping term for the mitochondrial OXPHOS complex assembly, as respiratory is just CI-IV)

We could either place 'mitochondrial respirasome assembly' under and change the defintion of GO:0033108 mitochondrial respiratory chain complex assembly to include supercomplexes formation:
"The aggregation, arrangement and bonding together of a set of components to form a mitochondrial respiratory chain complex or between respiratory chain complexes to form high-order structures."

or alternatively, if we want to group the assmebly of CI-CV and the respirasome, could make mitochondrial OXPHOS complex assembly term?

**Comment by ValWood** [comment]
Agreed, that's what the author of this paper said https://www.pombase.org/reference/PMID:39289458
Some genes are involved only in the assembly of supracomplexs and it shouldn't be a parent of the individual complex assembly

**Comment by raymond91125** [comment]
Note that we obsoleted GO:0070469 respirasome https://github.com/geneontology/go-ontology/issues/27253. We may consider reviving it with a tighter definition.

**Comment by hattrill** [comment]
@raymond91125 Think 'respirasome' should remain dead as it is not bringing anything to the party, apart from confusion. We need some sort of assembly BP as there are factors that are specific to this process (even if the supercomplex term is not useful)

**Comment by pgaudet** [comment]
Hi @raymond91125 

For the LD you propose [here:](https://github.com/geneontology/go-ontology/pull/29257/commits/feb8c0400d85898a7ea6bf88906c565c0e3ea444) 

is_a: GO:0065003 ! protein-containing complex assembly
intersection_of: GO:0022607 ! cellular component assembly
intersection_of: occurs_in GO:0005739 ! mitochondrion
intersection_of: results_in_assembly_of GO:0098803 ! respiratory chain complex


you should use 
intersection_of GO:0065003 ! protein-containing complex assembly and not GO:0022607 ! cellular component assembly

cellular component is to general, since it's the parent of  protein-containing complex  and cellular anatomical entity. 

Thanks, Pascale
